person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require_relative './nameable'
+
+class Person < Nameable
   def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
Your app is giving the error below (wrong number of arguments) since you have set the name variable to "Unknown" using the ` :`, kindly consider using the `=` sign to resolve this error. 
Also kindly ensure that all subsequent errors are resolved so that the `CapitalizeDecorator` and `TrimmerDecorator` classes also functions well. :pray: